Pr5

bsbrewer0 opened the pull request:
The results window file was changed so that the max LUFS/peak values from the platform were beside the actual LUFS/peak values from the files.

bsbrewer0 committed:
Small updates

--update the results window to make max lufs/peak values beside the actual value of the file.
--update to executable to match changes
--update to static results file to match the current code

bsbrewer0 committed:
Update main.py

drjeffjackson commented on SoundOff-seperate_files_classes/view_results_window.py:
```diff
@@ -225,16 +225,16 @@ def __init__(self, parent, names_selected, lufs_or_peak, peak, lufs, report_wind
             if lufs_or_peak == "LUFS Value":
-                lufs_label.grid(column=1, row=i, padx=5)
-                result_lufs.grid(column=2, row=i, padx=5)
+                lufs_label.grid(column=1, row=i, padx=5, sticky="e")
```
What do the sticky args do?

bsbrewer0 replied:
The sticky args change where the label appears within the column, so if the sticky argument is "e" (for east), it sticks to the right, rather than being centered within the column.

drjeffjackson replied:
Ah. Thanks!

drjeffjackson reviewed (commented):
SoundOff.exe cannot be run with ffmpeg, and ffmpeg cannot be pushed to a basic GitHub. Is there a good reason not to remove SoundOff.exe, .ico, and standards.db from the Application folder, since the user is going to have to unzip to get ffmpeg anyway?

bsbrewer0 commented:
No there isn't a good reason for why they're there, so I can remove them.

bsbrewer0 committed:
Delete SoundOff.exe

bsbrewer0 committed:
Delete SoundOff.ico

bsbrewer0 committed:
Delete standards.db

drjeffjackson committed:
Update README.md

Update GitHub Desktop instructions to be consistent with removal of the exe from the SoundOff_Application folder. Also simplify the web browser instructions by providing a direct link to the zip file.

drjeffjackson reviewed (approved):
I modified the SoundOff_Application README. If you agree with those changes, feel free to merge. Thank you again for all of the good work!!
